models/etcd/machine.go

```diff
@@ -258,8 +258,9 @@ RETRY:
 func (d *driver) machineQuery(ctx context.Context, q sabakan.Query) ([]*sabakan.Machine, error) {
 	var serials []string
 
+	qRemovedWithout := q.RemoveWithout()
```

Comment:
I could not read the intent of this code.
Does "without" mean the same as "empty"?

Reply:
I understood.
But I think this code is not readable.
Please rewrite this code or add a comment.